src/main/java/com/pretz/dyntreeserver/service/exceptions/UserCreationFailException.java

```diff
@@ -0,0 +1,10 @@
+package com.pretz.dyntreeserver.service.exceptions;
+
+import com.pretz.dyntreeserver.domain.User;
+
+public class UserCreationFailException extends RuntimeException {
+
+    public UserCreationFailException(User user) {
+        super("Creating user " + user.getName() + " in database failed.");
```

Comment:
same as in `IncorrectPasswordException `

Reply:
It looks like this exception is useless anyway, because there is no way for saving user to fail (Spring stuff handles it all as far as I understand). Or I could create some safeguard against creating the same user if there is one already in database.

Reply:
Done.